### PR TITLE
Start using LESS for styling, and customise bootstrap from the command line

### PR DESCRIPTION
**Pull request by txels**
Add a step to build styles from LESS files, by adding a build step to customise bootstrap from version-controlled sources.

### CODE REVIEW

**Comment by alexcouper** [comment]
Merging despite travis failure as I believe that will go away following merge of #49.

File: frontend.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
+CUSTOMISE BOOTSTRAP
+===================
+
+Install twitter bower and recess as npm modules, within the virtualenv:
+(bower for dependency management, and recess for LESS compilation)
+
+$ npm install -g bower
+$ npm install -g recess
+...both done by ./scripts/nodereqs.py
```
Comment (alexcouper) [inline frontend.txt:9]: This file is missing :) `git add` perhaps?

Comment (alexcouper) [inline frontend.txt:9]: Or maybe it just needs to be reworded to be `get_dependencies.py`

File: frontend.txt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Copy font-awesome font to our static font directory as post-install:
+$ cp components/font-awesome/font/* deploystream/static/font/
+
+
+INITIAL STEPS TO USE BOOTSTRAP AS A BASE
+========================================
+
+Make bootstrap files usable in our project:
+
+Symlink components/bootstrap/less to deploystream/static/less/bootstrap
```
Comment (alexcouper) [inline frontend.txt:27]: There was a wise man who once said the best form of documentation is an executable ;)

Comment (txels) [inline frontend.txt:27]: Ah no this is something to be done only once by me at the beginning, no scripting needed.
Re: font-awesome I think that is not probably going to be the final approach.
Eventually there will be some script update-dependencies that will update all (pip, npm, bower) in one go.

**Pull request merged**